cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/documentstore/DocumentStore.java

```diff
@@ -182,6 +173,30 @@ public static DocumentStore getInstance(File location, KeyProvider provider) thr
         }
     }
 
+    /**
+     * Get a reference to the {@link Database} object.
+     *
```

Comment:
These comments need `<p>` tags if we want this paragraphing to appear in the generated doc.